boot-microservices/src/main/java/org/ow2/proactive/boot/microservices/IAMStarter.java

```diff
@@ -26,6 +26,7 @@
 package org.ow2.proactive.boot.microservices;
 
 import java.io.*;
```

Comment:
Ideally you could remove the star imports.

Reply:
The star import is automatically generated (by idea) as I import all the classes under 'java.io' and 'java.util.concurrent'. The star allows for reducing the size of imports in such a case.